Admin/Ports/IDal.ts

```diff
@@ -0,0 +1,7 @@
+interface IDal {
```

Comment:
IApiSettingsRepository ?

Reply:
Why ApiSettings?

Reply:
I don't know if ApiSettings is the name, but let's think about more descriptive name than Dal

Reply:
Splitted to 
EnvironmentRepository and RequestRepository